packages/legend-application-studio/src/components/editor/editor-group/service-editor/__tests__/ServiceEditorTest.test.tsx

```diff
@@ -144,17 +145,22 @@ test(
 
     const bindingParmPairs = MOCK__editorStore.tabManagerState
       .getCurrentEditorState(ServiceEditorState)
-      .testableState.selectedSuiteState?.testStates[0]?.setupState.getBindingWithParamFromQuery();
+      .testableState.selectedSuiteState?.testStates[0]?.setupState.getBindingOrContentTypeWithParamFromQuery();
     expect(bindingParmPairs).toHaveLength(2);
     const firstPair = guaranteeNonNullable(
       guaranteeNonNullable(bindingParmPairs)[0],
     );
     const secondPair = guaranteeNonNullable(
       guaranteeNonNullable(bindingParmPairs)[1],
     );
-    expect(firstPair.binding.name).toBe('PersonBinding');
+
```

Comment:
please add a test specific to the PR here

Reply:
test is added